sigridci/sigridci.py

```diff
@@ -243,31 +243,34 @@ def __init__(self, options):
 
     def prepareUpload(self, sourceDir, outputFile):
         zipFile = zipfile.ZipFile(outputFile, "w", zipfile.ZIP_DEFLATED)
+        zipContents = []
         
         for root, dirs, files in os.walk(sourceDir):
             for file in sorted(files):
                 filePath = os.path.join(root, file)
                 if file != outputFile and not self.isExcluded(filePath):
                     relativePath = os.path.relpath(os.path.join(root, file), sourceDir)
                     uploadPath = self.getUploadFilePath(relativePath)
+                    zipContents.append(uploadPath)
                     if self.showContents:
                         log(f"Adding file to upload: {uploadPath}")
                     zipFile.write(filePath, uploadPath)
         
         zipFile.close()
         
-        self.checkUploadContents(outputFile)
+        self.checkUploadContents(outputFile, zipContents)
         
-    def checkUploadContents(self, outputFile):
+    def checkUploadContents(self, outputFile, zipContents):
         uploadSizeBytes = os.path.getsize(outputFile)
         uploadSizeMB = max(round(uploadSizeBytes / 1024 / 1024), 1)
         log(f"Upload size is {uploadSizeMB} MB")
         
         if uploadSizeMB > self.MAX_UPLOAD_SIZE_MB:
             raise Exception(f"Upload exceeds maximum size of {self.MAX_UPLOAD_SIZE_MB} MB")
             
-        if uploadSizeBytes < 50000:
-            log("Warning: Upload is very small, source directory might not contain all source code")
```

Comment:
Maybe we still want to give this warning as `elif` after checking whether there are contents at all?